nginx/nginx.conf.dev.template

```diff
@@ -10,7 +10,40 @@ server {
     listen [::]:80;
     listen 80;
     server_name ${NGINX_HOST};
-    keepalive_timeout 65;
+
```

Comment:
Now this file is almost equivalent to `nginx-ssl.conf.template`. The only difference is this block:

```
    location /docs {
        auth_basic "OpenEduHub";
        auth_basic_user_file /etc/nginx/.htpasswd;
        try_files $uri $uri/ =404;
        index index.html;
    }
```
Can we simply drop the dev template and use `nginx-ssl.conf.template` for dev and prod?

Reply:
Good point, thanks for pointing out. I removed the /docs part, because it is unused. Test in dev was succesful